spec/factories/organisations.rb

```diff
@@ -15,5 +15,9 @@
   factory :organisation do
     name { "LONDON SCITT" + rand(1000000).to_s }
     sequence(:org_id)
+
+    trait :with_user do
+      users { [find_or_create(:user)] }
```

Comment:
Not so sure about the `find_or_create` here (sorry if I suggested it should be this).

Reason being I think it's more common that, by default, each organisation gets one user. If we want one user to be shared across organisations, it's my feeling we should have to do that explicitly.

Reply:
Ok makes sense